Silicon/CommonSocPkg/Library/VtdPmrLib/IntelVtdPmr.c

```diff
@@ -22,6 +23,7 @@
   @return protected low memory alignment.
 **/
 UINT32
+STATIC

```

Comment:
For internal function, I remember  it is not necessary to add "STATIC" although I don't know if really have difference.

Reply:
It is recommended to use STATIC for internal function to avoid potential conflicts on same function names in different libs.